src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-right: $cards-distance;
+  max-width: 200px;
+  padding: 0 16px;
+  font-weight: 400;
```

Comment:
It's a default value
![image](https://user-images.githubusercontent.com/91411692/223071763-cef6538f-ca89-4096-9ca4-cf39cd9b656f.png)
![image](https://user-images.githubusercontent.com/91411692/223071786-15869429-11e3-4729-bb66-fceb9b475490.png)

```suggestion
```